packages/react-components/react-jsx-runtime/README.md

```diff
@@ -35,15 +40,27 @@ const renderButton_unstable = (state: ButtonState) => {
 };
 ```
 
-In case you're using typescript or any modern javascript transpiler, you can use [JSX import source](https://www.typescriptlang.org/tsconfig#jsxImportSource) feature instead of depending on declaring the runtime on every file
+### JsxRuntime classic
+
+> 💡 NOTE: no need to specify `@jsx` pragma because our api exports `createElement` - same name symbol as React exports
+
+```tsx
+/** @jsxRuntime classic */
 
-In TSC case you can simply add this do `tsconfig.json` file:
+import { createElement } from '@fluentui/react-jsx-runtime';
+import { assertSlots } from '@fluentui/react-utilities';
 
-```json
```

Comment:
removed this completely as we should not encourage to use this pragma within every jsx in user land